vignettes/ExtendingGenomicRanges.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "**Extending Genomic Ranges**"
+author: "Michael Lawrence, Bioconductor Team"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+package: GenomicRanges
+vignette: >
+  %\VignetteIndexEntry{Extending Genomic Ranges}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+output:
```

Comment:
Please change output to include TOC: 

output:
  BiocStyle::html_document:
    number_sections: yes
    toc: yes
    toc_depth: 4